### PR TITLE
answer the comments for the release review

### PR DESCRIPTION
**Pull request by jianhong**
## PR checklist

- [x] This comment contains a description of changes (with reason).
- [x] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/nf-core/hicar/tree/master/.github/CONTRIBUTING.md)
- [ ] If necessary, also make a PR on the nf-core/hicar _branch_ on the [nf-core/test-datasets](https://github.com/nf-core/test-datasets) repository.
- [x] Make sure your code lints (`nf-core lint`).
- [x] Ensure the test suite passes (`nextflow run . -profile test,docker --outdir <OUTDIR>`).
- [x] Usage Documentation in `docs/usage.md` is updated.
- [x] Output Documentation in `docs/output.md` is updated.
- [x] `CHANGELOG.md` is updated.
- [x] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

**Comment by jianhong** [comment]
I will stop here untill the PR is merged. And then I will merge the template 2.9. After that, I will continue again.

File: nextflow_schema.json
Comment (SPPearce) [inline nextflow_schema.json]:
```suggestion
                    "description": "create cooler files for maps or not",
```

File: modules/local/atacreads/getscale.nf
Comment (SPPearce) [inline modules/local/atacreads/getscale.nf]: Don't need this now.

File: modules/local/atacreads/getscale.nf
Comment (SPPearce) [inline modules/local/atacreads/getscale.nf]: Don't use this anymore.
I'd also say that typically you just put 
`def prefix   = task.ext.prefix ? "${task.ext.prefix}" : "${meta.id}"`
and then people can use the meta.id or not when they define the prefix.

**Review by SPPearce — CHANGES_REQUESTED** [review]

File: modules/local/atacreads/getscale.nf
Comment (jianhong) [reply modules/local/atacreads/getscale.nf]: Got it.

**Comment by jianhong** [comment]
@SPPearce Did I fixed the comments for current changes? If yes, could you please approve this change? Thank you.

**Comment by jianhong** [comment]
@mirpedrol Do you have time to go through this PR? I want to merge the new template to remove the lint errors.

**Review by SPPearce — APPROVED** [review]

**Comment by SPPearce** [comment]
Oh, linting is failing, but I think that is just the template changes.

File: modules/local/atacreads/getscale.nf
Comment (mirpedrol) [inline modules/local/atacreads/getscale.nf:1]: I am a bit confused by this module, I think you can do this with a map in the main nextflow code.

**Review by mirpedrol — APPROVED** [review]
Only one last comment, otherwise looks good to me :)

Linting is failing because it's running with nf-core/tools v2.9 but the pipeline is still in v2.8. If you make sure that the linting passes locally with nf-core/tools v2.8 it's enough for me and I am ok with ignoring this test for now

File: modules/local/atacreads/getscale.nf
Comment (jianhong) [reply modules/local/atacreads/getscale.nf:1]: Yes, I agree with you for that I can simply use multiple steps in the main nextflow code. My idea is that this function will be used by multiple subworkflows, to reduce the code repetition, I put it as a module.

**Comment by jianhong** [comment]
> Only one last comment, otherwise looks good to me :)
> 
> Linting is failing because it's running with nf-core/tools v2.9 but the pipeline is still in v2.8. If you make sure that the linting passes locally with nf-core/tools v2.8 it's enough for me and I am ok with ignoring this test for now

I will merge the template once this one got merged. The lint issue should be fixed in next PR. Thank you.

**Comment by jianhong** [comment]
@SPPearce @mirpedrol I'am deeply appreciated if any of you could click the Merge button.

**Pull request merged**
